pkg/bpf/bpf.go

```diff
@@ -68,6 +69,7 @@ func NewBpfLoader(config *options.BpfConfig) *BpfLoader {
 }
 
 func (l *BpfLoader) StartAdsMode() (err error) {
+	var ve *ebpf.VerifierError
```

Comment:
This should be var ve ebpf.VerifierError, otherwise the below &ve will panic

Reply:
> This should be var ve ebpf.VerifierError, otherwise the below &ve will panic

The function we finally call is 
```shell
func ErrorWithLog(source string, err error, log []byte, truncated bool) *VerifierError {
       ... ...
}
```
So, this ve should be *ebpf.VerifierError, And actually when I change  *ebpf.VerifierError to  ebpf.VerifierError, it will panic
![Uploading image.png…]()